src/Model/Api/Response/Catalog/BuilderTemplateResponse.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tweakwise\TweakwiseJs\Model\Api\Response\Catalog;
+
+use Magento\Framework\Data\Collection\AbstractDb;
+use Magento\Framework\Model\Context;
+use Magento\Framework\Model\ResourceModel\AbstractResource;
+use Magento\Framework\Registry;
+use Tweakwise\TweakwiseJs\Api\Data\Api\Response\Catalog\BuilderTemplateResponseInterface;
+use Tweakwise\TweakwiseJs\Model\Api\Response;
+use Tweakwise\TweakwiseJs\Model\Api\Type\TemplateType;
+use Tweakwise\TweakwiseJs\Model\Api\Type\TemplateTypeFactory;
+
+class BuilderTemplateResponse extends Response implements BuilderTemplateResponseInterface
+{
+    /**
+     * @param Context $context
+     * @param Registry $registry
+     * @param TemplateTypeFactory $templateTypeFactory
+     * @param AbstractResource|null $resource
+     * @param AbstractDb|null $resourceCollection
+     * @param array $data
+     */
+    public function __construct(
+        Context $context,
+        Registry $registry,
+        protected readonly TemplateTypeFactory $templateTypeFactory,
+        AbstractResource $resource = null,
+        AbstractDb $resourceCollection = null,
```

Comment:
Can you make it php 8.4 compatible?
`?AbstractResource $resource = null,
  ?AbstractDb $resourceCollection = null,`